packages/server/src/api/routes/tests/queries/generic-sql.spec.ts

```diff
@@ -61,7 +66,7 @@ describe.each([
     // We re-fetch the datasource here because the one returned by
     // config.api.datasource.create has the password field blanked out, and we
     // need the password to connect to the database.
-    const ds = await dsProvider.datasource()
+    const ds = await dsProvider
```

Comment:
This await on a variable seems not natural to me. But not sure what would be the best naming

Reply:
Yeah, I thought it looked weird as well. I can instead pass in the function, rather than the promise, but I believe this comes with the downside that the containers are not started in parallel, and will lead to each test waiting on its respective container at the start of the tests (if the container is not already running).